src/client/js/create.js

```diff
@@ -458,8 +459,15 @@ async function sendStudyData() {
         body: JSON.stringify(createStudyDatas),
       });
       const result = await request.json();
-      console.log(result, "request 결과");
+      console.log(result.message); 
```

Comment:
console.log는 pr올리실때는 빼주시고 올려주세요 잊어버리신거 같아서 남겨놓습니다
```suggestion
      console.log(result.message); 
```